src/cdk-experimental/combobox/combobox-panel.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+
+@Directive({
+  selector: 'ng-template[cdkComboboxPanel]',
+  exportAs: 'cdkComboboxPanel',
+  host: {
+    'role': 'combobox'
```

Comment:
The panel here wouldn't be the element w/ `role="combobox"`, it would be the trigger.

Reply:
Yeah that shouldn't have been there my mistake.